CHANGELOG.md

```diff
@@ -1,16 +1,27 @@
-# Changlog
+# Changelog
+All notable changes to this project will be documented in this file.
 
-## [New Releases here](https://github.com/kickstarter/rack-attack/releases)
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
```

Comment:
Can we not include this sentence for now?
I don't think we're there yet.

Thank you.

Reply:
No problem.